datarepo-clienttests/src/main/java/testscripts/IngestFile.java

```diff
@@ -3,12 +3,8 @@
 import bio.terra.datarepo.api.RepositoryApi;
 import bio.terra.datarepo.api.ResourcesApi;
 import bio.terra.datarepo.client.ApiClient;
-import bio.terra.datarepo.model.BillingProfileModel;
-import bio.terra.datarepo.model.DatasetSummaryModel;
-import bio.terra.datarepo.model.DeleteResponseModel;
-import bio.terra.datarepo.model.FileLoadModel;
-import bio.terra.datarepo.model.FileModel;
-import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.*;
```

Comment:
Nit: I think we have a team norm against * imports. I think you can turn this off in IntelliJ.
![Screen Shot 2020-07-24 at 12 19 36 PM](https://user-images.githubusercontent.com/12446128/88412742-0af13080-cda8-11ea-9f9b-599828813a06.png)

Reply:
omg I keep missing this I'm an idiot